src/v2/api/index.md

```diff
@@ -951,29 +951,29 @@ type: api
 
 - **See also:** [Components](../guide/components.html)
 
-## Options / Composition
+## Opciones / Composición
 
 ### parent
 
 - **Tipo:** `Vue instance`
 
-- **Details:**
+- **Detalles:**
 
-  Specify the parent instance for the instance to be created. Establishes a parent-child relationship between the two. The parent will be accessible as `this.$parent` for the child, and the child will be pushed into the parent's `$children` array.
+  Especifica la instancia padre para la instancia que será creada. Establece la relación padre-hijo entre ambas. El padre será accesible como `this.$parent` para el hijo, y el hijo será insertado en el arreglo `$children` del padre.
 
-  <p class="tip">Use `$parent` and `$children` sparingly - they mostly serve as an escape-hatch. Prefer using props and events for parent-child communication.</p>
+  <p class="tip">Use `$parent` y `$children` con moderación. Prefiera el uso de _props_ y eventos para la comunicación entre padre y hijo.</p>
 
 ### mixins
 
 - **Tipo:** `Array<Object>`
 
-- **Details:**
+- **Detalles:**
 
-  The `mixins` option accepts an array of mixin objects. These mixin objects can contain instance options like normal instance objects, and they will be merged against the eventual options using the same option merging logic in `Vue.extend()`. e.g. If your mixin contains a created hook and the component itself also has one, both functions will be called.
+  La opción `mixins` acepta un arreglo de objetos _mixin_. Estos objetos pueden contender opciones de instancia, así como instancias de objetos normales, y luego serán combinados con las eventuales opciones utilizando la misma lógica de combinación que en `Vue.extend()`. Por ejemplo, sí su `mixin` contiene el _hook_ _created_ y su componente también lo tiene, ambas funciones serán invocadas.
 
-  Mixin hooks are called in the order they are provided, and called before the component's own hooks.
+  Los _hooks_ de _mixin_ son invocadas en el orden que son proporcionados, e invocadas antes que los _hooks_ de la componente.
```

Comment:
- Use the same gender for hooks. `Los`, `invocados`. 
- Typo `de la componente`, change to `del componente`.